datalad_debian/bootstrap_builder.py

```diff
@@ -28,15 +28,44 @@
 @build_doc
 class BootstrapBuilder(Interface):
     """Bootstrap a build environment
+
+    This command bootstraps a (containerized) build environment, such as a
+    Singularity container, based on an existing builder configuration, such as
+    a Singularity recipe.
+
+    The execution of this command might require administrative privileges and
+    could prompt for a sudo password, for example in order to build a
+    Singularity image. The resulting bootstrapped build environment will be
+    placed inside of a 'envs/' subdirectory of a 'builder/' dataset-
+
+    The following directory tree illustrates this with an example. The
```

Comment:
Style nit: "illustrates" and "with an example" are redundant. Only one is necessary.

Reply:
thx!